client/app/create-recipes/create-recipes-controller.js

```diff
@@ -0,0 +1,34 @@
+angular.module('wtf.create-recipes', [])
+  .controller('CreateRecipesController', function($scope, $window, Auth, Recipe) {
+
+    $scope.recipe = {};
+
+    $scope.$watch(Auth.isAuth, function(authed) {
+      if (authed) {
+        $location.path('/create-recipes');
+      } else {
+        $location.path('/');
+      }
+    }, true);
+
+    $scope.root = 
+
+    $scope.submit = function(ingredient) {
```

Comment:
Comment this out for now; our submit method will be a bit different as we're taking both a title and ingredients